src/styles/blocks/stars.css

```diff
@@ -0,0 +1,36 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(../../images/star-active.svg);
+}
+
```

Comment:
You can combine the selectors with common styles. Also, path in url should be written with quotes
https://developer.mozilla.org/en-US/docs/Web/CSS/background-image